DatadogRUM/Sources/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -50,6 +50,8 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
     let viewName: String
     /// The start time of this View.
     let viewStartTime: Date
+    /// The load time of this View.
+    var viewLoadingTime: TimeInterval?
```

Comment:
or just `private`, no?
```suggestion
    private(set) var viewLoadingTime: TimeInterval?
```